src/globalComponents.spec.ts

```diff
@@ -1,9 +1,3 @@
 import { assert } from 'chai';
 import * as React from 'react';
-import "./globalComponents";
 
-describe('globalComponents', ()=>{
```

Comment:
@adamabeshouse Here is a test I don't think we need.  I mean, there needs to be a line drawn somewhere.  A good one I've seen is basically, test logic, don't test declaration.   I think exposing a function is declaration.